### DISCUSSION TITLE
Ruler Dimensions

### DISCUSSION THREAD

**Discussion by Dandy201** [The-OpenROAD-Project/OpenROAD#6129] [Q&A] [answered]
Hi, what dimensions does the ruler use? For example, what does 6921 x 6921 mean? Is it in nm?
![image](https://github.com/user-attachments/assets/76a1a910-b500-40d3-8191-899aa08425b2)

**Comment by gadfort** [comment, marked as answer]
The units for the ruler (and other GUI elements) is um. However, it looks like you may have turned on `Show DBU` in which case the units are the database units. Can you can toggle that setting in the Options menu:
![image](https://github.com/user-attachments/assets/f28a7d5b-3525-4ca7-8e17-1cedac37a909)

**Reply by Dandy201** [reply]
Alright thanks!
